From 421f8ee7f3959d1e3819fad4fb385cc612f7efbf Mon Sep 17 00:00:00 2001
From: Mika Laitio <lamikr@gmail.com>
Date: Sat, 6 Jan 2024 08:13:00 +0200
Subject: [PATCH 1/2] remove AQL_PROFILE warning

old rocm-5.3.1 can be easily patched not to require
the proprietary AQL profiler library.
(Newer rocprofiler versions use it more extensively in
many places of the code)

Signed-off-by: Mika Laitio <lamikr@gmail.com>
---
 cmake_modules/env.cmake | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/cmake_modules/env.cmake b/cmake_modules/env.cmake
index 2e9613b..1e9150a 100644
--- a/cmake_modules/env.cmake
+++ b/cmake_modules/env.cmake
@@ -120,7 +120,7 @@ if ( "${ROCM_ROOT_DIR}" STREQUAL "" )
   message ( FATAL_ERROR "ROCM_ROOT_DIR is not found." )
 endif ()
 
-find_library ( FIND_AQL_PROFILE_LIB "libhsa-amd-aqlprofile64.so" HINTS ${CMAKE_INSTALL_PREFIX} PATHS ${ROCM_ROOT_DIR})
-if (  NOT FIND_AQL_PROFILE_LIB )
-  message ( FATAL_ERROR "AQL_PROFILE not installed. Please install AQL_PROFILE" )
-endif()
+#find_library ( FIND_AQL_PROFILE_LIB "libhsa-amd-aqlprofile64.so" HINTS ${CMAKE_INSTALL_PREFIX} PATHS ${ROCM_ROOT_DIR})
+#if (  NOT FIND_AQL_PROFILE_LIB )
+#  message ( FATAL_ERROR "AQL_PROFILE not installed. Please install AQL_PROFILE" )
+#endif()
-- 
2.48.1

